bool stopped
float32 rot_vel
---
bool success